components/EventBanner.tsx

```diff
@@ -0,0 +1,37 @@
+// CSS
+import styles from "../styles/components/EventBanner.module.scss";
+// Config
+import * as config from "../config.yaml";
```

Comment:
Wait does this need to be `* as config`? alternatively it might be better to import the specific fields you need like I did in the navbar

Reply:
I mean if we're taking the contextKey as a prop and selecting the prop key from the yaml, we do need to `import *` since it has to have all fields available to be able to select the correct one, no?